scripts/cnode-helper-scripts/env

```diff
@@ -1031,6 +1031,21 @@ test_koios() {
   [[ ${ENABLE_KOIOS} = 'Y' && -n ${KOIOS_API} && $(curl -Isf -m 5 ${KOIOS_API}/tip | head -1) = *"200 OK"* ]] || unset KOIOS_API
 }
 
+check_cncli_versions() {
+  if [[ $(command -v cncli) ]] ; then
```

Comment:
I think I mentioned it but we should also check if a cncli sync service is deployed for this instance. This command could be used to check status. Exit code 0 if running. So
```bash
systemctl is-active --quiet ${CNODE_VNAME}-cncli-sync.service
```
So line could then be modified to:
```bash
if command -v cncli >/dev/null && systemctl is-active --quiet ${CNODE_VNAME}-cncli-sync.service; then
```